src/libbitcoind.cc

```diff
@@ -236,6 +236,28 @@ NAN_METHOD(GetBestBlockHash) {
   }
 }
 
+NAN_METHOD(GetNextBlockHash) {
+
+  if (args.Length() < 1 || !args[0]->IsString()) {
+    return NanThrowError("Usage: bitcoind.getNextBlockHash([blockhash])");
```

Comment:
The first argument is a string?  e.g. `Usage: bitcoind.getNextBlockHash(blockhash)`